src/bridges/tasklist.ts

```diff
@@ -88,11 +126,6 @@ export const TaskListBridge = new BridgeExtension<
   
   ul[data-type="taskList"] li > div {
     flex: 1 1 auto;
-  }
-  
-  ul[data-type="taskList"] li ul li,
-  ul[data-type="taskList"] li ol li {
```

Comment:
Why did we delete this? can we validate if needed or how it impacts?

Reply:
It breaks the styling for tasklist items, tested and didn't see any bad effect